lib/aws/rails/sqs_active_job/executor.rb

```diff
@@ -9,7 +9,7 @@ module SqsActiveJob
       class Executor
         DEFAULTS = {
           min_threads: 0,
-          max_threads: Concurrent.processor_count,
+          max_threads: Integer(Concurrent.available_processor_count || Concurrent.processor_count),
```

Comment:
Can either of these ever be nil? Why is Integer wrapping processor_count too?

Reply:
`Concurrent.available_processor_count` can be nil, but `Concurrent.processor_count` will not.  `Concurrent.available_processor_count` returns a float, which needs to be converted to an Integer for max_threads.